src/lib/DatePicker.svelte

```diff
@@ -127,6 +127,25 @@
     const maxDate = new Date(max.getFullYear(), max.getMonth(), max.getDate())
     return date >= minDate && date <= maxDate
   }
+  function yearKeydown(e: KeyboardEvent) {
+    if (e.key === 'ArrowUp' || e.key === 'ArrowLeft') {
+      setYear(year - 1)
+      e.preventDefault()
+    } else if (e.key === 'ArrowDown' || e.key === 'ArrowRight') {
+      setYear(year + 1)
+      e.preventDefault()
+    }
+  }
```

Comment:
Previously you were able to use `Left/Right` to change the month when tabbed to the year. Would be nice to keep that in my opinion (they basically would correspond to the arrow buttons on the popup)

Reply:
restored left/right behavior to change month when year control is focused. I'm not sure which behavior is more intuitive - this would be a good one for hallway usability.

Reply:
Left/right isn't used for dropdowns anyway, so should be good